web/src/lib/components/pages/record.tsx

```diff
@@ -96,11 +96,10 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
     this.onProgress = this.onProgress.bind(this);
   }
 
-  private refillSentenceCache() {
-    return this.props.api.getRandomSentences(CACHE_SET_COUNT)
-      .then(newSentences => {
-        this.sentenceCache = this.sentenceCache.concat(newSentences);
-      });
+  private async refillSentenceCache() {
+    const newSentences = await this.props.api.getRandomSentences(CACHE_SET_COUNT)
+    this.sentenceCache = this.sentenceCache.concat(newSentences);
+    return this.sentenceCache;
```

Comment:
this return value is not declared and never used, so IMO nothing should be returned here (even though I think technically it did also return this value before)